### ISSUE TITLE
Make the lights out section more attractive

### ISSUE THREAD

**Issue by abishekvashok** [fossasia/gci15.fossasia.org#529]
Hey guys the light's out section is'nt having that much standard as the rest of the page. The color also looks not that much great :) I hope someone will resolve this and the mentors would agree with this issue
![untitled-2](https://cloud.githubusercontent.com/assets/8947010/12443693/b3523aee-bf7f-11e5-958c-2fe82315dbef.png)

**Comment by jigyasa-grover** [comment]
@Abhi2424shek Would you like to work on this ?

**Comment by abishekvashok** [comment]
@jig08 sure but I completed most of my tasks on GCI for the website! But if you want I may work it out, but really I posted it for other students so that they could earn points! I may work if you want. I would be happy too :)

**Comment by mananwason** [comment]
:+1:

**Comment by jigyasa-grover** [comment]
@Abhi2424shek No issues, We shall let someone else work on it.

**Comment by abishekvashok** [comment]
Ok I just wanted to give me fellows a chance to work on @jig08 #social commitment. Like me they also want to be a grand prize winner right? :) - laughing out loud

**Comment by mananwason** [comment]
Hey man. Calm down !! This is open to everyone. If you think you can solve the issue, feel free to do so, if you think that you can't then someone else will come and do it.

**Comment by SaptakS** [comment]
@Abhi2424shek  at the end of the day its completely your own decision.. So, take your call if you want to do it.. Else someone will..

**Comment by jmurzaku** [comment]
i can work on this task if you don't want @Abhi2424shek

**Comment by JamMurz** [comment]
@Abhi2424shek i have this issue fixed but havent commited it yet tell me if you still want to do it

**Comment by mariobehling** [comment]
Please keep the style of the headings (same font, same font size)

**Comment by abishekvashok** [comment]
hey @mananwason I never got angry and I won't. I am laughing typing all these! :) Wanna have a click from my web cam in my uniform! :-) [Hey is there a laughing emoji I can use in GitHub) ? :)

**Comment by yasoob** [comment]
Hey @Abhi2424shek the short-code for the laughing emoji is `:laughing:` but I don't think that you should use it too much.

**Comment by abishekvashok** [comment]
not laughing - smiling

**Comment by mananwason** [comment]
@Abhi2424shek Then use `:smile:`

**Comment by abishekvashok** [comment]
:smile: Thanks for teaching out now I can have drop downs on pressing the semi-colon key ':' and typing a letter. Thanks for that I have only used to be posting in a forum called planet-cricket where I used to be developing bats(May provide a link and it was years old however the quality was great) for cricket 07. There were strict manners there and also it's my second coding experience after the one by Intel in which my idea is among the five which represents the state in the national level competition.
